tests/targets_preparer_test.py

```diff
@@ -65,3 +66,12 @@ def testPrepareTargets_whenIPv6AssetReachCIDRLimit_raiseValueError(
 
     with pytest.raises(ValueError, match="Subnet mask below 112 is not supported."):
         assert any(targets)
+
+
+def testPrepareTargets_whenUrlCantBeParsed_returnEmptyTargets(
+    scan_bad_message: message.Message,
+) -> None:
+    targets = targets_preparer.prepare_targets(scan_bad_message)
+
+    with pytest.raises(NotImplementedError):
+        assert any(targets) is False
```

Comment:
Can you make `any(targets)` more explicit? It's hard to tell what it's checking for.